src/app/js/admin/progress/Progress.js

```diff
@@ -16,22 +16,16 @@ import { PENDING } from '../../../../common/progressStatus';
 import { polyglot as polyglotPropTypes } from '../../propTypes';
 
 const renderProgressText = props => {
-    const { progress, target, symbol } = props;
-    if (!progress || !target) {
+    const { progress, target, symbol, label, p: polyglot } = props;
+    if (!progress) {
         return null;
     }
 
-    if (symbol) {
-        return (
-            <p>
-                {progress} / {target} {symbol}
-            </p>
-        );
-    }
-
     return (
         <p>
-            {progress} / {target}
+            {`${progress}${target ? ` / ${target}` : ''}${
```

Comment:
It's not very readable. After that I don't have a better solution. Except to return it in a function, but I don't know if it's better